recipes/ValueBasedJsonCodecLog/src/Main.purs

```diff
@@ -46,9 +46,8 @@ main = do
   log $ "Encoding the example value:"
   log $ stringify $ encode entireRecordCodec exampleValue
 
-
 exampleJson :: Json
-exampleJson = unsafePartial $ fromRight $ jsonParser
+exampleJson = unsafePartial $ (\(Right json) -> json) $ jsonParser
```

Comment:
`fromRight` is no longer a partial function https://pursuit.purescript.org/packages/purescript-either/5.0.0/docs/Data.Either#v:fromRight

Reply:
I considered making a standalone function for this, something like
```haskell
fromRight :: Partial => forall a b. Either a b -> b
fromRight (Right x) = x
```
but I opted for the minimal change. I'm definitely open to suggestions here; this one in particular felt like one of the more questionable changes.

Reply:
I think people have typically been doing something like `either (\_ -> unsafeCrashWith "got Left: bad parser") identity`. It's a bit longer, but then it at least provides a somewhat clearer error message.

Reply:
a1f97fe